pipeline/processor/postprocessor.py

```diff
@@ -308,7 +310,7 @@ def _format_semantic_lists(self, df: pd.DataFrame) -> List[List[str]]:
         df["type"] = df["path"].apply(lambda x: x[-1])
         # Sometimes label is called ExtraCharSpan, replace it with label
         df["type"] = df["type"].replace(
-            {"ExtraCharSpan": "Lbl", "ParagraphSpan": "LBody"}
+            {"ExtraCharSpan": "Lbl", "ParagraphSpan": "LBody", "Span": "Lbl"}
```

Comment:
Span elements complicate things a lot with Adobe and there's nothing general. But I noticed that assuming span elements are labels if they belong to list elements works in most cases. Another upstream fix.

Reply:
🙌🏼